### PR TITLE
Hardware recommendation edits

### PR DESCRIPTION
**Pull request by portertech**
A few updates from Engineering and some cloud guidance.

Objectives:

- Capture difference between minimal and recommended
- Emphasize more is better (over-provisioning is good for monitoring)
- Emphasize sustained disk IOPS
- Some Cloud guidance

### CODE REVIEW

**Review by apaskulin — COMMENTED** [review]
🙌 It's great to have more content on this page! I've included some suggested style edits in this PR: https://github.com/sensu/sensu-docs/pull/1372

**Review by apaskulin — APPROVED** [review]

File: content/sensu-go/5.5/installation/recommended-hardware.md [hunk 1]
```diff
@@ -11,67 +11,99 @@ menu:
...
+may find that additional CPU cores (8+) are necessary.
 
-Each check that is executed results in writes occurring in the backend.
-When provisioning storage hardware, a good rule of thumb is to have twice as
-many iops as you expect to have checks executing per second. Don't forget to
-include keepalives in this calculation; each agent executes a keepalive
-every 20 seconds. So in a cluster of 100 agents, you can expect those agents
-to consume 10 write iops for keepalives.
+Every executed Sensu check results in storage writes. When
+provisioning storage, a good rule of thumb is to have twice as many
```
Comment (cwjohnston) [inline content/sensu-go/5.5/installation/recommended-hardware.md]:
```suggestion
provisioning storage, a good guideline is to have twice as many
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.5/installation/recommended-hardware.md (hunk 1 above)
Comment (apaskulin) [reply content/sensu-go/5.5/installation/recommended-hardware.md]: Fixed in https://github.com/sensu/sensu-docs/pull/1371/commits/fac248bb22988a0f5347388f32fac7a6135a701b

**Pull request merged**